src/site/_collections/authors.js

```diff
@@ -150,14 +150,6 @@ module.exports = (collections) => {
     authors[key] = author;
   });
 
-  // Only complain that authors are invalid if we've got any posts *at all*
```

Comment:
is it ok to remove this checkaltogether, or should it be disabled only in "isolate" mode?

Reply:
@MichaelSolati wdyt? Is there any danger to removing this check?

Reply:
I don't think there is any danger in removing it, bit I think it would be safer to conditionally ignore it if you want. Are there any authors that this brakes for?